### ISSUE TITLE
Instalar en una web

### ISSUE THREAD

**Issue by fercol06** [qborreda/guia-tv#1]
Hola, 
¿como puedo instalarlo en una web? Cuando lo ejecuto desde el navegador no carga nada y se desconfigura.
Un saludo.

**Comment by qborreda** [comment]
Claro. De hecho, no debería estar esto aquí, debería ser un repo privado.
Es un prototipo de la que ahora mismo está ya colgada en http://www.elmundo.es/television/programacion-tv/
aunque forzando con datos de prueba, con fecha 14 de Julio de 2014 y para sólo seis canales.

Básicamente, tienes que instalar los componentes con npm, y montar el servidor con gulp, que además empaqueta el javascript, preprocesa las css desde sus scss, y demás ..

Si lo que buscas es aprender javascript, este no es un buen repo, y de hecho, tengo que eliminar esto de aquí .. aunque todo el código sea mío ..
